common/src/main/proto/TransportMessages.proto

```diff
@@ -38,6 +38,8 @@ enum MessageType {
   GET_REDUCER_FILE_GROUP_RESPONSE = 15;
   UNREGISTER_SHUFFLE = 16;
   UNREGISTER_SHUFFLE_RESPONSE = 17;
+  BATCH_UNREGISTER_SHUFFLES = 21;
```

Comment:
Use unused numbers to ensure these RPCs won't be treated as old RPCs.

Reply:
Done!